src/app/components/scrollBar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+const ScrollBarContainer = styled.div({
+  overflowY: 'hidden',
+  scrollbarGutter: 'stable',
+  ':hover': {
+    overflowY: 'auto',
+  },
+  '&::-webkit-scrollbar': {
+    width: 8,
+    height: 10,
+    background: 'transparent',
+  },
+  '&::-webkit-scrollbar-thumb': {
+    width: 8,
+    maxHeight: 10,
+    borderRadius: 24,
+    background: 'var(--white-800, rgba(255, 255, 255, 0.20))',
+  },
+});
+
+interface Props {
+  children: ReactNode;
+}
+function ScrollBar({ children }: Props): JSX.Element {
+  return (
+  <ScrollBarContainer>
+    {children}
+  </ScrollBarContainer>
+  );
+}
+
+export default ScrollBar;
```

Comment:
I'm not sure using a react container is the best way to do this.

if all we need to do is centralise the hidden scrollbars CSS, then let's follow the same pattern as in theme/index.ts with typographies.

then usage will be same as for apply typography styles to any styled component, and you don't need to refactor all the JSX which wants hidden scrollbars

Reply:
OK I'm currently working on only adding the scrollbar CSS rather than a separate component. However, the scrollable screens do need one refactoring even with the CSS which would be around the margin. since we need to use `scrollbar-gutter: stable both edges` property which allocates symmetric spacing to both sides of the screen so that the layout does not shift when the scroll bar is visible . The width of the scrollbar gutter is the same as the width of the scrollbar so in our case 8px. This would mean there would be a 8px margin on both sides on all screens

Reply:
yep, I think that's still preferable to refactoring with asymmetric left/right margins